acinclude.m4

```diff
@@ -2097,7 +2097,13 @@ AC_DEFUN([PHP_PROG_RE2C],[
   fi
   case $php_cv_re2c_version in
     ""|invalid[)]
-      AC_MSG_WARN([You will need re2c 0.13.4 or later if you want to regenerate PHP parsers.])
+      msg="You will need re2c 0.13.4 or later if you want to regenerate PHP parsers."
```

Comment:
This message should probably be changed for the error case.

Reply:
Yes, for sure. Fixed in the next commit to make these two actions more distinct and meaningful...